src/containers/EvmCrossChainRecoveryCoin/PolygonFormHotWallet.tsx

```diff
@@ -0,0 +1,144 @@
+import { Form, FormikHelpers, FormikProvider, useFormik } from 'formik';
+import { Link } from 'react-router-dom';
+import * as Yup from 'yup';
+import { Button, FormikTextfield } from '~/components';
+
+const validationSchema = Yup.object({
+  userKey: Yup.string().required(),
+  feeAddress: Yup.string().required(),
+  gasLimit: Yup.number()
+    .typeError('Gas limit must be a number')
+    .integer()
+    .positive('Gas limit must be a positive integer')
+    .required(),
+  maxFeePerGas: Yup.number().required(),
+  maxPriorityFeePerGas: Yup.number().required(),
+  recoveryDestination: Yup.string().required(),
+  walletContractAddress: Yup.string().required(),
+  walletPassphrase: Yup.string().required(),
+  bitgoFeeAddress: Yup.string(),
+}).required();
+
+export type PolygonFormProps = {
+  onSubmit: (
+    values: PolygonFormValues,
+    formikHelpers: FormikHelpers<PolygonFormValues>
+  ) => void | Promise<void>;
+};
+
+type PolygonFormValues = Yup.Asserts<typeof validationSchema>;
+
+export function PolygonFormHotWallet({ onSubmit }: PolygonFormProps) {
+  const formik = useFormik<PolygonFormValues>({
+    onSubmit,
+    initialValues: {
+      userKey: '',
+      feeAddress: '',
+      gasLimit: 500000,
+      maxFeePerGas: 20,
+      maxPriorityFeePerGas: 10,
```

Comment:
Please use proper values here and also can we add these in config?

Reply:
What should be the values here ?
I have used the same values used in Non Bitgo Recovery and Unsigned sweep.